### ISSUE TITLE
sparcc returning missing as correlation result

### ISSUE THREAD

**Issue by asizemore** [microbiomeDB/microbiomeComputations#24]
Would this be expected? Found on qa
<img width="830" alt="Screen Shot 2024-07-15 at 2 03 20 PM" src="https://github.com/user-attachments/assets/fc0af364-cb97-433b-9881-8c5fcee1f1ab">

**Comment by asizemore** [comment]
Found again on qa today

<img width="931" alt="Screen Shot 2024-08-12 at 5 25 18 PM" src="https://github.com/user-attachments/assets/4f706eb4-a857-41fb-9294-8cddf7362fda">


Rserve says
```
Error in if (correlationCoef < 0) { :
  missing value where TRUE/FALSE needed
```

**Comment by d-callan** [comment]
fixed in rserve 8.1.3
